docker/Dockerfile

```diff
@@ -51,6 +51,16 @@ RUN pip3 install pybullet stable-baselines3 && \
     ros-${ROS_DISTRO}-pointgrey-camera-driver && \
     rm -rf /var/lib/apt/lists/* /root/.cache/pip/
 
+# Install eigen
+RUN mkdir eigen && \
```

Comment:
@dgerblick can't we just get this through APT? seems like overkill

Reply:
Didn't realize apt had eigen, oops